packages/graphiql/package.json

```diff
@@ -69,18 +69,20 @@
     "browserify-shim": "3.8.14",
     "chai": "4.2.0",
     "chai-subset": "1.6.0",
+    "common-shakeify": "^0.6.0",
```

Comment:
This package has a large number of contributors on npm for its download count, I'm concerned that it may increase our attack surface area (though it is a devDependency, which lessens the risk a little).

Reply:
removed, yeah wasnt doin much for us yet anyways